apps/crn-server/scripts/export-entity.ts

```diff
@@ -114,15 +121,27 @@ function getController(entity: Entity) {
   const assetDataProvider = new AssetSquidexDataProvider(userRestClient);
 
   const controllerMap = {
-    users: new Users(userDataProvider, assetDataProvider),
-    'research-outputs': new ResearchOutputs(
+    user: new Users(userDataProvider, assetDataProvider),
+    'research-output': new ResearchOutputs(
       researchOutputDataProvider,
       researchTagDataProvider,
       externalAuthorDataProvider,
     ),
-    'external-authors': new ExternalAuthors(squidexGraphqlClient),
-    events: new Events(squidexGraphqlClient, eventsRestClient),
+    'external-author': new ExternalAuthors(squidexGraphqlClient),
+    event: new Events(squidexGraphqlClient, eventsRestClient),
+    lab: new Labs(squidexGraphqlClient),
   };
 
   return controllerMap[entity];
-}
+};
+
+const transformRecords = <T extends keyof EntityResponses>(
```

Comment:
So we are still doing the transform step rather than adding it to the entity?

Reply:
yeh at least here it was easier